ui/src/api/api.ts

```diff
@@ -72,6 +88,176 @@ export async function getEntities(arxivId: string, getAllEntities?: boolean) {
+          sentence: toLegacyRelationship(
+            deduped.relationships.sentence,
+            'sentence'
+          ),
+          nickname_sentences: deduped.relationships.nickname_sentences.map(
+            (n) => toLegacyRelationship(n, 'sentence')
+          ),
+          defining_formula_equations: (
+            response.sharedSymbolData[disambiguatedId]?.defining_formula_equations || []
+          ).map((s) => toLegacyRelationship(s, 'equation')),
+          definition_sentences: (
+            response.sharedSymbolData[disambiguatedId]?.definition_sentences || []
+          ).map((s) => toLegacyRelationship(s, 'sentence')),
+          snippet_sentences: (
+            response.sharedSymbolData[disambiguatedId]?.snippet_sentences || []
+          ).map((s) => toLegacyRelationship(s, 'sentence')),
+        },
+      };
+      return symbol;
+    } else if (isTerm(deduped)) {
+      const term: Term = {
+        id: deduped.id,
+        type: 'term',
+        attributes: {
+          bounding_boxes: deduped.attributes.bounding_boxes,
+          name: deduped.attributes.name,
+          source: 'tex-pipeline', // hardcoded since most everything comes out of the tex pipeline
+          tags: [],
+          term_type: deduped.attributes.term_type,
+          definitions: deduped.attributes.definitions,
+          definition_texs: deduped.attributes.definition_texs,
+          sources: deduped.attributes.sources,
+          snippets: deduped.attributes.snippets,
+        },
+        relationships: {
+          sentence: toLegacyRelationship(
+            deduped.relationships.sentence,
+            'sentence'
+          ),
+          definition_sentences: (
+            deduped.relationships.definition_sentences || []
+          ).map((s) => toLegacyRelationship(s, 'sentence')),
+          snippet_sentences: (
+            deduped.relationships.snippet_sentences || []
+          ).map((s) => toLegacyRelationship(s, 'sentence')),
+        },
+      };
+      return term;
+    }
+    // There's a new kind of entity that hasn't been implemented in the UI.
+    throw "Unknown entity type";
+  });
+  return {
+    data: entities,
+  };
+}
+
+
+/**
+ * This API returns a compacted representation of the paper's entities and their relationships.
+ * Certain fields of Symbol entity data are pulled into a separate `sharedSymbolData` map,
+ * which is organized by the Symbols' `attributes.disambiguated_id` value.
+ *
+ * NOTE: Currently, this function passes the response through a transform function to make
+ * it compatible with the existing UI code.
+ *
+ * @param arxivId arXiv ID of the viewed paper
+ * @param getAllEntities `true` retrieves entities of all types, `false` retrieves only citations
+ * @returns
+ */
+export async function getDedupedEntities(arxivId: string, getAllEntities?: boolean) {
+  const params = getAllEntities ? {
+    type: ENTITY_API_ALL
+  } : {};
```

Comment:
Checking my understanding: does setting `type` to `{}` results in the API returning the default set of entities?